.travis.yml

```diff
@@ -2,9 +2,10 @@ language: dart
 dart:
   - 1.24.3
   - stable
+before_install:
+  - ./pub2get.sh
 script:
-  - pub get
   - pub run dart_dev format --check
   - pub run dart_dev analyze
-  - pub run dart_dev test
+  - pub run test
```

Comment:
Why this change? Should we specify the unit test file to run?

Reply:
`pub run test` seemed cleaner compare to  `pub run dart_dev test --release -p chrome`. Both of them work, I can change it `pub run dart_dev test --release -p chrome` if that we prefer.

Reply:
The reason I asked is I noticed there is a generated runner in this repo. Since we're not specifying a test file to run it seems like it's going to run all the test files it finds which will result in all of the tests running twice.

I assumed ddev test was configured to run just the generated runner, but it appears there is no configuration for it in the dev.dart file. So when we were running ddev test, I'm not sure what it was doing.

Either way, it seems like it would be nice to at least specify the generated runner as the suite we want to run, whether thats on the command line when running pub run test or it is in the dev.dart test config. 🤷‍♂

Reply:
You know what? The generated runner doesn't end in `test.dart` so it is never run. I think we could just delete that file and call it good.